src/StackExchange.Redis/APITypes/StreamGroupInfo.cs

```diff
@@ -33,4 +35,14 @@ internal StreamGroupInfo(string name, int consumerCount, int pendingMessageCount
     /// The Id of the last message delivered to the group.
     /// </summary>
     public string? LastDeliveredId { get; }
+
+    /// <summary>
+    /// Total number of entries the group had read.
+    /// </summary>
+    public int? EntriesRead { get; }
```

Comment:
Should this be int64?

Reply:
Updated. Does it make sense to update `lag` for the same reason? Although that should unusual to have a consumer group being created and reading messages from the beggining on very large streams.

Reply:
Updated. Does it make sense to update `lag` for the same reason? Although that should be unusual to have a consumer group being created and reading messages from the beginning on very large streams.

Reply:
it sounds *unusual*, but if it is theoretically possible, we should probably make the `Lag` 64-bit too, I guess.

Reply:
Fixed on c57fbfbd129a9b4aca88b81e4d44a6b4e329739d and c57fbfbd129a9b4aca88b81e4d44a6b4e329739d.